src/Json.Schema.ToDotNet/DataModelGeneratorSettings.cs

```diff
@@ -82,6 +82,12 @@ public class DataModelGeneratorSettings
         /// </summary>
         public bool GenerateCloningCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating what C# type to generate for Json type
+        /// <code>Integer</code>.
+        /// </summary>
+        public string GenerateIntegerAs { get; set; }
```

Comment:
>string [](http://example.com/codeflow?start=15&length=6)

Should this be a string or should it be an enum?

Reply:
good point, changed to enum.